recipes-connectivity/networkmanager/networkmanager-0.9.2.0/use-correct-data-in-nl-cb.patch

```diff
@@ -0,0 +1,13 @@
+Index: NetworkManager-0.9.2.0/src/nm-netlink-monitor.c
+===================================================================
+--- NetworkManager-0.9.2.0.orig/src/nm-netlink-monitor.c
++++ NetworkManager-0.9.2.0/src/nm-netlink-monitor.c
+@@ -248,7 +248,7 @@ nlh_setup (struct nl_sock *nlh,
+ {
+ 	int err;
+ 
+-	nl_socket_modify_cb (nlh, NL_CB_MSG_IN, NL_CB_CUSTOM, event_msg_recv, cb_data);
++	nl_socket_modify_cb (nlh, NL_CB_MSG_IN, NL_CB_CUSTOM, event_msg_recv, nlh);
```

Comment:
Candidate for backport if NetworkManager is not upgraded before, this is a lingering segfault in 32bit as well.